src/content/docs/mlops/integrations/openai-integration.mdx

```diff
@@ -75,24 +82,144 @@ openai.Completion.create(
     temperature=0 
 )
 ```
+</Step>
+</Steps>
+ </Collapser>
+
+
+
+<Collapser 
+    id="install-with-node" 
+    title="Install with Node.js"
+  >
+<Steps>
+<Step>
+### Install OpenAI [#install-openai-node]
+
+Use your favorite package manager and run:
+
+```
+npm install @newrelic/openai-observability openai
+```
+</Step>
+<Step>
+### Getting started [#getting-started-node]
+
+Call `monitorOpenAI` with the `openAI`instance. 
+
+Here is some example code:
+
+```
+import { Configuration, OpenAIApi } from 'openai';
+import { monitorOpenAI } from '@newrelic/openai-observability';
+
+const configuration = new Configuration({
+  apiKey: 'OPENAI_API_KEY',
+});
+
+const openAIApi = new OpenAIApi(configuration);
+
+monitorOpenAI(openAIApi, {
+  applicationName: 'MyApp',
+  newRelicApiKey: 'NEW_RELIC_LICENSE_KEY',
+});
+
+const response = await openAIApi.createChatCompletion({
+  model: 'gpt-3.5-turbo',
+  messages: [{ role: 'user', content: 'What is Observability?' }],
+});
+
+console.log(response.data.choices[0].message);
+```
+
+Make sure that you add module type to package.json:
+
+```
+"type": "module"
+```
+</Step>
+
+<Step>
+### Initialization options [#modify-code]
+
+```
+export interface MonitorOpenAIOptions {
+  /**
+   * Your application name in New Relic, must provided.
+   */
+  applicationName: string;
+  /**
+   * API key with insert access used to authenticate the request.
+   * For more information on creating keys, please see:
+   * https://docs.newrelic.com/docs/insights/insights-data-sources/custom-data/introduction-event-api#register
+   */
+  newRelicApiKey?: string;
+  /**
+   * Optional host override for event endpoint.
+   */
+  host?: string;
+  /**
+   * Optional port override for trace endpoint.
+   */
+  port?: number;
+}
+```
+</Step>
+<Step>
+### Environment variables [environment-variables-node]
 
-## EU account users [#EU-account-users]
+You can use your configured environment variable for initialization options:
+
+`NEW_RELIC_LICENSE_KEY - API`: Key with insert access used to authenticate the request.
+
+`NEW_RELIC_INSERT_KEY`: Same as API key.
+
+`EVENT_CLIENT_HOST`:  Optional host override for events endpoint.
+</Step>
+<Step>
+### Test 
```

Comment:
i'm not really sure what this step does. I think it means 'test that it was set up properly' but i would clarify. i think this is the 'trigger some data and find it in our UI' step

Reply:
Done